database/sampleDataQueries.sql

```diff
@@ -32,14 +32,14 @@ VALUES
 
 INSERT INTO User (FirstName, LastName, EmailAddress, Password, Address, ZipCode, DistancePreference, LastUpdated)
 VALUES
-  ('Esther', 'Lin', 'linhsin@oregonstate.edu', 'password', NULL, 10001, NULL, '2022-01-27 12:32:05');
+  ('Esther', 'Lin', 'linhsin@oregonstate.edu', '$2a$10$UEYuur6YO08R8mgcHL8wP.DDtExNajJD/m8plsHFSjy45oNDdEUgC', NULL, 10001, NULL, '2022-01-27 12:32:05');
 
 INSERT INTO PetBreed (TypeOfAnimal, Breed)
 VALUES
   ('Cat', 'American ShortHair'),
   ('Cat', 'Domestic ShortHair'),
   ('Cat', 'Other'),
   ('Dog', 'Golden Retriever'),
-  ('Dog', 'Mixed'),
-  ('Dog', 'Other'),
+  ('Dog', 'German Shepherd'),
```

Comment:
I have two GSDs, so I couldn't resist adding this when I saw it! If you want me to make this change in a different PR, just let me know!